dist/src/main/resources/api/cluster/components.yaml

```diff
@@ -93,6 +93,55 @@ schemas:
         $ref: "components.yaml#/schemas/CompletedChange"
       pendingChange:
         $ref: "components.yaml#/schemas/TopologyChange"
+      routing:
+        $ref: "components.yaml#/schemas/RoutingState"
+
+  RoutingState:
+    description: The routing state, i.e. where requests are sent to and how messages are correlated
+    type: object
+    required:
+      - version
+      - activePartitions
+      - messageCorrelation
+    properties:
+      version:
+        type: integer
+        format: int64
+        description: The version of the routing state
+        example: 1
+      activePartitions:
+        type: array
+        description: The list of active partitions that accept round-robin requests
+        example: [1, 2, 3]
+        items:
+          type: integer
+          format: int32
+      messageCorrelation:
+        $ref: "components.yaml#/schemas/MessageCorrelation"
+
+
+  MessageCorrelation:
+    description: The message correlation strategy
+    oneOf:
+      - $ref: "components.yaml#/schemas/MessageCorrelationHashMod"
+    discriminator:
```

Comment:
:+1: TIL

Reply:
Not suuuper ergonomic because we still have to know about this value and set it when building the response but otherwise quite nice.